comp/core/secrets/secretsimpl/fetch_secret.go

```diff
@@ -121,17 +121,22 @@ func (r *secretResolver) fetchSecret(secretsHandle []string) (map[string]string,
 	secrets := map[string]secrets.SecretVal{}
 	err = json.Unmarshal(output, &secrets)
 	if err != nil {
+		for _, sec := range secretsHandle {
+			r.tlmSecretResolveError.Inc("unmarshal", sec)
+		}
```

Comment:
Do we need to send a metric for each handle in this case? 
Could it be that for one call of `fetchSecret` we fail to unmarshal and for the next one to not fail? 

mostly asking because we might just need t increment once in this scenario

Reply:
I can use a different metric for that error yes 👍